src/app/tests/integration/Dockerfile.initiator

```diff
@@ -1,4 +1,21 @@
-from chip-cirque-device-base
+#
+#   Copyright (c) 2020 Project CHIP Authors
```

Comment:
2021

Is this license notice really required?

Reply:
I added it to others for consistency, it was in some, but not others.